cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -131,14 +143,6 @@
   <!-- Router Configuration -->
   
   <property>
-    <name>router.address</name>
```

Comment:
Well, as far as I can tell, this is something only used by our scripts under Linux (Docker). I don't really think it should be in here if it can be avoided, as I don't think CDAP is really using it. I will discuss with @wolf31o2.